pytonlib/client.py

```diff
@@ -671,12 +672,15 @@ async def try_locate_tx_by_incoming_message(self, source, destination, creation_
 
         for shard_data in shards['shards']:
             shardchain = shard_data['shard']
-            for b in range(3):
```

Comment:
3 was not enough for some cases and transaction wouldn't be found. Increased to 10.

Reply:
Hi @jdziworski-bc, thank you for your PR. I think there are these points to consider:
1. Usually the distance of blocks with out_tx and in_tx becomes > 3 when the network experiences a high load and the message queue contains many entries. In that case, the distance might be even greater than 10.
2. Under high load the network splits into shards.
3. Under high load lite servers are usually busy and response time increases significantly.

Let's say the network has 10 shards, for each shard we lookup + read transactions of 10 blocks. In total, it results in 200 requests to the lite server in the worst case. It is not the desired outcome in toncenter.com/api/v2 because it puts an additional load on liteservers.
As an option, I suggest adding the parameter `scan_blocks_count` with default value 3 and you can adjust it for your use case. WDYT?